src/main/java/core/basesyntax/service/operations/ReturnOperation.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax.service.operations;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+import java.util.List;
+
+public class ReturnOperation implements OperationHandler {
+
+    @Override
+    public void getOperation(List<FruitTransaction> transactions) {
+        List<FruitTransaction> transactionsSelected = transactions.stream()

```

Comment:
Consider using the Strategy pattern more effectively by having the `OperationStrategy` call the appropriate `OperationHandler` based on the operation type. Filtering transactions by operation type should not be done within the `ReturnOperation` class.